interface/orders/orders_results.php

```diff
@@ -208,17 +208,41 @@ function QuotedOrNull($fld) {
 <script type="text/javascript" src="../../library/dialog.js"></script>
 <script type="text/javascript" src="../../library/textformat.js"></script>
 <script type="text/javascript" src="../../library/js/jquery-1.9.1.min.js"></script>
-
+<?php call_required_libraries(array("jquery-min-3-1-1","bootstrap", "iziModalToast")); ?>
 <script language="JavaScript">
 
 
-// This invokes the find-procedure-type popup.
+// This invokes the find-procedure-type popup. Old popup style iframe replaced with iziModal. 
 var ptvarname;
+// function sel_proc_type(varname) {
+//  var f = document.forms[0];
+//  if (typeof varname == 'undefined') varname = 'form_proc_type';
+//  ptvarname = varname;
+//  dlgopen('types.php?popup=1&order=' + f[ptvarname].value, '_blank', 800, 500);
+// }
+
+//This invokes the find-procedure-type popup in iziModal iframe.
 function sel_proc_type(varname) {
  var f = document.forms[0];
  if (typeof varname == 'undefined') varname = 'form_proc_type';
  ptvarname = varname;
- dlgopen('types.php?popup=1&order=' + f[ptvarname].value, '_blank', 800, 500);
+ // dlgopen('types.php?popup=1&order=' + f[ptvarname].value, '_blank', 800, 500); // This calls the old style popup
+ event.preventDefault();
+    console.log(event.target.href);
+    $("#modal-iframe").iziModal({
+    title: 'Find procedure',
+    subtitle: 'Select procedure from the table below', headerColor: '#88A0B9',
+    iframe: true,
+    iframeHeight: 500,
+    width: 700,
+    height: 500,
```

Comment:
@mshubhankar good work done here. Just some few stuffs to be done. izi headers on the other modals

Reply:
I'm sorry I couldn't fully understand. Which other modal are you exactly taking about ? Is it the Add button or Add top level modal ?